app/parse.py

```diff
@@ -12,7 +19,89 @@ class Course:
     name: str
     short_description: str
     course_type: CourseType
+    duration: str
+    modules: int
+    topics: int
+
+
+all_courses = []
+
+
+def get_course_type(course_type: str) -> CourseType:
+    if course_type.lower() in ["full-time", "повний день"]:
```

Comment:
it's better to not hardcode such string, the best approach is to create constants or constant list